pytradfri/command.py

```diff
@@ -119,25 +118,6 @@ def _merge(
                 b_dict[key] = value
         return b_dict
```

Comment:
We can remove `_merge` too I think, right?

Reply:
Yes, it was being used in combine only.